src/Components/Categories.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable quotes */
+import React from "react";
+import Header from "./Header";
+import Button from "./Shared/Button";
+
+const Categories = () => (
+  <>
+    <Header />
```

Comment:
- [ ] We would be repeating ourselves here by importing the `Header` component here and also importing it in the App component. It is repetitive. Kindly remove this import here and import the Header only one in the App component. It would be available throughout the app.